indico/modules/rb/models/rooms.py

```diff
@@ -400,6 +400,11 @@ def sprite_position(self):
         sprite_mapping = _cache.get('rooms-sprite-mapping')
         return sprite_mapping.get(self.id, 0) if sprite_mapping else None  # placeholder at position 0
 
+    @property
+    def is_blocked(self):
```

Comment:
This sends one query whenever accessed, so using this in a loop over many rooms is not a good idea.